jboss/container/wildfly/launch/os/node-name/tests/openshift-node-name.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+source $BATS_TEST_DIRNAME/../added/launch/openshift-node-name.sh
+
+@test "init_node_name: NODE_NAME defined the value is expected to be adopted" {
+  unset JBOSS_NODE_NAME
+  NODE_NAME=12345678912345678912345 init_node_name
+  [ "${#JBOSS_NODE_NAME}" -lt 24 ]
+  [ "$JBOSS_NODE_NAME" = "12345678912345678912345" ]
+}
+
+@test "init_node_name: NODE_NAME defined the value has to be shortened to 23 characters" {
+  unset JBOSS_NODE_NAME
+  NODE_NAME=abcdefghijklmnopqrstvuwxyz init_node_name
+  [ "${#JBOSS_NODE_NAME}" -eq 23 ]
+  [ "$JBOSS_NODE_NAME" = "abcdefghijklmn-7d3817b1" ]
+}
```

Comment:
One more test that verifies the expected behaviour would be nice too, something like (untested, but you get the idea)
``
@test "init_node_name: truncation collision" {
  unset JBOSS_NODE_NAME
  NODE_NAME=abcdefghijklmnopqrstvuwxyz init_node_name
  [ "${#JBOSS_NODE_NAME}" -eq 23 ]
  [ "$JBOSS_NODE_NAME" = "abcdefghijklmn-7d3817b1" ]
  prev=$JBOSS_NODE_NAME
  unset JBOSS_NODE_NAME
  NODE_NAME=abcdefghijklmnopqrstvuwxy1 init_node_name
  [ "${#JBOSS_NODE_NAME}" -eq 23 ]
  echo NN: ${JBOSS_NODE_NAME}
  [ "$JBOSS_NODE_NAME" = "${prev}" ]
  
}